articles/security-center/security-center-app-services.md

```diff
@@ -20,7 +20,7 @@ ms.author: rkarlin
 # Protect App Service with Azure Security Center
 This article helps you use Azure Security Center to monitor and protect your applications running on top of App Service.
 
-App Service enables you to build and host web applications in the programming language of your choice without managing infrastructure. App Service offers auto-scaling and high availability, support both Windows and Linux, as well as automated deployments from GitHub, Visual Studio Team Services or any Git repository. 
+App Service enables you to build and host web applications in the programming language of your choice without managing infrastructure. App Service offers auto-scaling and high availability, support both Windows and Linux, as well as automated deployments from GitHub, Azure Repos or any Git repository. 
 
```

Comment:
@mlearned I don't think this should be Azure Repos? Not clear what it's referring to. Shouldn't there be a mention of Azure Pipelines if it's about automated deployments or is the point that you can automate deployments from any repo wherever it is?

Reply:
@kasangi  i can see how this is confusing, we often talk about "deployments from the repo (Git)" so in this case it would indeed be ... "GitHub, Azure Repos, or any Git repository."
As you are saying Azure Pipelines is also involved (but kind of implied here) 

GitHub uses Webhooks (or you could use Jenkins or any number of ways) .. in our side we'd have both Azure Repos and Azure Pipelines as part of the story :).    

Clear as mud?    Essentially here they are leaving of the orchestration tooling piece.

It wouldn't be evil to say GitHub, Azure Pipelines, ...  implying the Repo either. but it sounds strange since they are talking about version control provider.

Reply:
should be GitHub, Azure Repos, or any Git repo 

So before we might have generally referred to things as "VSTS" but now instead of talking about the overall service (Azure DevOps) the guidance is to refer to the specific service .. in this case Azure Repos